src/EventService.Data.Provider.MsSql.Ef/Migrations/20230126193800_InitialTables.cs

```diff
@@ -157,6 +157,85 @@ private void CreateEventFilesTable(MigrationBuilder migrationBuilder)
       });
   }
   
+  private void CreateGroupsTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbGroup.TableName,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        Name = table.Column<string>(nullable: false),
+        MaxGuests = table.Column<int>(nullable: true),
+        Description = table.Column<string>(nullable: false),
+        IsActive = table.Column<bool>(nullable: false),
+        CreatedBy = table.Column<Guid>(nullable: false),
+        CreatedAtUtc = table.Column<DateTime>(nullable: false),
+        ModifiedBy = table.Column<Guid>(nullable: true),
+        ModifiedAtUtc = table.Column<DateTime>(nullable: true) 
+      },
+      constraints: table =>
+      {
+        table.PrimaryKey($"PK_{DbGroup.TableName}", ec => ec.Id);
+      });
+  }
+  
+  private void CreateOccasionsTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbOccasion.TableName,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        EventId = table.Column<Guid>(nullable: false),
+        Name = table.Column<string>(nullable: false),
+        MaxGroup = table.Column<int>(nullable: true),
+        Description = table.Column<string>(nullable: true),
+        IsActive = table.Column<bool>(nullable: false),
+        CreatedBy = table.Column<Guid>(nullable: false),
+        CreatedAtUtc = table.Column<DateTime>(nullable: false),
+        ModifiedBy = table.Column<Guid>(nullable: true),
+        ModifiedAtUtc = table.Column<DateTime>(nullable: true) 
+      },
+      constraints: table =>
+      {
+        table.PrimaryKey($"PK_{DbOccasion.TableName}", ec => ec.Id);
+      });
+  }
+  
+  private void CreateGroupsEventsUsersTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbGroupEventUser.TableName,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),

```

Comment:
You don't need Id here. The pair of event and group is unique so use them as clustered PK as written below